_episodes/03-sharing.md

```diff
@@ -19,69 +19,69 @@ keypoints:
 - "`diff` is a Git verb for viewing the difference between an edited file and the file's most recent commit"
 ---
 
+## The power of sharing
+
+The real power of Git lies in being able to share your work with others and in being able to work collaboratively. The best way to do this is to use a remote hosting platform. For this lesson, we are using GitHub. Let's log in there now. 
+
 ## Create a repository on GitHub
 
-When we have logged in to GitHub, we can create a new repository by clicking the **+** icon in the upper-right corner of
+Once we have logged in to GitHub, we can create a new repository by clicking the **+** icon in the upper-right corner of
 any page then selecting **New repository**. Let's do this now.
 
-* Create a new repository
-* Give it the name `hello-world`
+![The GitHub website top navigation with the 'add new ...' button. ](../fig/github-repo-new.png)
+
+* Click "New Repository"
+
+Clicking `New Repository` will take you to a creation page with different options. For this workshop, we are not using any of the options available. 
+
+* Name your repository "hello-world."
+
+![The 'create a new repository' form on GitHub](../fig/github-repo-new-setup.png) 
 
-GitHub will ask if you want to add a README.md, license or a `.gitignore` file. Do not do any of that for now.
```

Comment:
I don't agree with removing this line 
> GitHub will ask if you want to add a README.md, license or a `.gitignore` file. Do not do any of that for now.

If someone includes any files on the remote when it's created, it will cause an error when the local repo is pushed.  Since we're setting up learners to succeed at one thing at a time, it's important that the remote repository is completely empty.

Reply:
I think you might not have noticed that I had a line about that above the screenshot, but I think you might be right that the clarity of that sentence is a good repetition. I will add it back in. 

This is what I did have in the lesson, but I think the two sentences you point out in this comment are worth keeping indeed. 
On line 35 of this lesson, I have: 

> Clicking `New Repository` will take you to a creation page with different options. For this workshop, we are not using any of the options available.

Reply:
@emcaulay, Yes, I did lose that sentence.

Reply:
I am in favour of adding the removed line back in too. Any files added to the GitHub repository when it is created here will cause trouble later in the lesson, when the learner first tries to sync their local and remote versions, and I think the redundancy of including both these sentences is worth it to further guard against that.